test/coverlet.core.tests/Instrumentation/MethodInfoExtensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+
+namespace Coverlet.Core.Instrumentation.Tests
+{
+    public static class MethodInfoExtensions
```

Comment:
I'll much prefer this as a single method in the relevant class file

Reply:
Do you mean that this method must be placed in `InstrumenterTests` class?

Reply:
See my previous comment about using extension methods sparingly. The method defined in here is fine as a helper within the test class that uses it. If it's used in multiple places feel free to create a static class (`TestUtils.cs`?) and put it and all other utility methods in there

Reply:
Maybe we'll leave it like this?